python/tests/sql/test_function.py

```diff
@@ -1177,3 +1177,9 @@ def test_hausdorffDistance(self):
         actual_default = actual_df_default.take(1)[0][0]
         assert expected == actual
         assert expected == actual_default
+
+    def test_st_coord_dim(self):
+
+        point_df = self.spark.sql("SELECT ST_GeomFromWkt('POINT(7 8 6)') AS geom")
```

Comment:
This test does not actually test `ST_CoordDim` python DataFrame API. Please carefully read the test case examples in this file.

Reply:
Can you please elaborate on what you mean? I can't understand.